lib/removePipelineExtras.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = removePipelineExtras;
+
+function removePipelineExtras(gltf) {

```

Comment:
Okay, sounds good! I'll add those tests.

I'm thinking of something like the following case for adding extras:
```
"animations": {
    "animation_0": {
        "channels": [
            {
                "sampler": "animation_0_scale_sampler",
                "target": {
                    "id": "Bone",
                    "path": "scale"
                }
            }
        ],
        "parameters": {
            "TIME": "animAccessor_0"
        }
    }
}
```
In this case, we want to add extras to `animation_0`, the single channel in `channels`, and that channel's target. However, we wouldn't add extras to the `parameters` object in `animation_0`. So I guess I'm just confused about how I would differentiate between an object like `parameters` versus a glTF object like `target`.